source/includes/_plugins.md

```diff
@@ -0,0 +1,265 @@
+# Edge Wallet Plugins
+
+Edge Wallet plugins are a way to include additional functionality into Edge
+Wallet on iOS and Android. It is currently used to buy Bitcoin, Ethereum and
+Litecoin via Simplex.
+
+The Wallet Plugin API provides a subset of the entire Edge SDK which allows the
+plugin to access the currently logged in user's wallet to request sends and
+receives.
+
+These pages serve as an introduction on writing your own plugins. You only need
+basic web development skills to build your own plugin. If you can write HTML,
+CSS and Javascript, you can easily write a plugin and run it on Android and iOS.
+
+## How it works
+
+Edge plugins are just single page HTML files, with all the resources "compiled"
+in. That means that the Javascript libraries, stylesheets and whatever else are
+all included in one monolithic HTML file.
+
+The code is loaded by the Edge Wallet into a sandboxed webview. A bridge is
+created between the webview and the native-app via the
+[`edge-libplugin`][libplugin] package.  Using the [`edge-libplugin`][libplugin]
+package, plugins can easily:
+
+- Read and write encrypted data
+- Request Sends
+- Generate receive addresses for coins and tokens
+
+## Creating a plugin
+
+```javascript
+git clone https://github.com/teneighty/edge-plugin-skeleton
+cd edge-plugin-skeleton
+npm install
+```
+
+To get started you can clone the skeleton project. This already includes
+[`edge-libplugin`][libplugin] as a dependency. If you look in the `package.json`
+you'll also find the command `edge`. This command is responsible for taking
+your code and creating the monolithic HTML file which the native app will load.
+
+Note that each plugin is expected to have a `manifest.json` which will tell the
+react native app where and how to display the plugin.
+
+## Load plugin into Edge
+
+First you'll have to have the Edge Wallet building locally. Please follow the
+instructions in the Edge Wallet [README][edge-readme].
+
+Once you are able to build Edge Wallet, you can make your plugin a dependency
+and add it to the `plugins` array at the bottom of the `package.json`. 
+
```

Comment:
Maybe a little bit more detail here: npm / git repo should be added to `dependencies` and a property with a value of an array (is the key arbitrary?) should be added with an array member matching the dependency module.